ipn/store.go

```diff
@@ -26,6 +26,13 @@ const (
 	// particular), but right now Unix daemons run with a single
 	// node-global state. To keep open the option of having per-user state
 	// later, the global state key doesn't look like a username.
+	//
+	// As of 2022-10-21, it has been superseded by profiles and is no longer
+	// written to disk. It is only read at startup when there are no profiles,
+	// to migrate the state to the "default" profile.
+	// The existing state is left on disk in case the user downgrades to an
+	// older version of Tailscale that doesn't support profiles. We can
+	// remove this in a future release.
 	GlobalDaemonStateKey = StateKey("_daemon")
```

Comment:
Maybe rename this to `LegacyGlobalDaemonStateKey` or something like that to make it clearer that no new uses should be added?